modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * assembly_installing_pipelines.adoc
+
+[id="proc_installing-pipelines-operator-via-yaml.adoc_{context}"]
+= Installing OpenShift Pipelines from the OperatorHub using the CLI
+
+You can install OpenShift Pipelines from the OperatorHub using the CLI.
```

Comment:
OpenShift Pipelines operator

Reply:
Fixed in https://github.com/openshift/pipelines-docs/pull/57/commits/dd55a60f9f465192dfb3f4fd371d87d76df1c7db